app/controllers/likes_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
         format.json { render :json => @like.as_api_response(:backbone), :status => 201 }
       end
     else
-      render :nothing => true, :status => 422
+      render text: I18n.t("javascripts.failed_to_like"), status: 422
```

Comment:
> but we also need to retranslate it

I would have restructured the translation files in a lot of PRs if we wouldn't have to retranslate the stuff. And using JavaScript translations in the Rails code is easy. The other way round isn't.